RemotingSample/Assets/Prefabs/AlphaTestSpheres.prefab

```diff
@@ -0,0 +1,224 @@
+%YAML 1.1
```

Comment:
Are you using this prefab? This goes along with the Black0Alpha materials I mentioned

Reply:
looks like these nodes are not necessary here in this sample, removed all the relevant stuff